src/main/java/com/cdancy/bitbucket/rest/domain/admin/DetailedUser.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.cdancy.bitbucket.rest.domain.admin;
+
+import com.cdancy.bitbucket.rest.BitbucketUtils;
+import com.cdancy.bitbucket.rest.domain.common.Error;
+import com.cdancy.bitbucket.rest.domain.common.ErrorsHolder;
+import com.google.auto.value.AutoValue;
+import org.jclouds.javax.annotation.Nullable;
+import org.jclouds.json.SerializedNames;
+
+import java.util.List;
+
+@AutoValue
+public abstract class DetailedUser implements ErrorsHolder {
+
+    @Nullable
+    public abstract String name();
+
+    @Nullable
+    public abstract String emailAddress();
+
+    @Nullable
+    public abstract Integer id();
+
+    @Nullable
+    public abstract String displayName();
+
+    @Nullable
+    public abstract Boolean active();
+
+    @Nullable
+    public abstract String slug();
+
+    @Nullable
+    public abstract String type();
+
+    @Nullable
+    public abstract String directoryName();
+
+    @Nullable
+    public abstract Boolean deletable();
+
+    @Nullable
+    public abstract Long lastAuthenticationTimestamp();
+
+    @Nullable
+    public abstract Boolean mutableDetails();
+
+    @Nullable
+    public abstract Boolean mutableGroups();
+
+    DetailedUser() {
+    }
+
+    @SerializedNames({ "name", "emailAddress", "id", "displayName", "active", "slug", "type", "directoryName",
+            "deletable", "lastAuthenticationTimestamp", "mutableDetails", "mutableGroups", "errors" })
+    public static DetailedUser create(final String name, final String emailAddress, final int id,
+                                      final String displayName, final boolean active, final String slug,
+                                      final String type, final String directoryName, final boolean deletable,
+                                      long lastAuthenticationTimestamp, boolean mutableDetails,
+                                      final boolean mutableGroups, final List<Error> errors) {
+
+        return new AutoValue_DetailedUser(BitbucketUtils.nullToEmpty(errors), name, emailAddress, id, displayName,
+            active, slug, type, directoryName, deletable, lastAuthenticationTimestamp, mutableDetails, mutableGroups);
+    }
+}
```

Comment:
Lets move the changes here into the `User` domain object instead of creating something new.

Reply:
But what about the errors? I mean, there was instruction somewhere that domain objects that are used to create request can't be an error handlers.